src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -127,6 +143,7 @@ Complete the following prerequisites before you remove one or more nodes from a
       ```
       kubectl -n rook-ceph exec deployment/rook-ceph-operator -- ceph status
       ```
+      **Note**: It is not recommended to use versions of Rook Ceph earlier than v1.4.0.
```

Comment:
```suggestion
      **Note**: It is no longer recommended to use versions of Rook Ceph earlier than v1.4.0.
```

Reply:
@paigecalvert just a nit. Otherwise, all shows good for me.

Reply:
I took a look at this suggestion, and I recommend leaving this as "not required" rather than "no longer required" because the phrase "no longer" assumes that the user has knowledge about requirements from the past.

We try to keep the docs "timeless", without any reference to past or future. (Google has a good description of "timeless documentation" here, if you are interested: https://developers.google.com/style/timeless-documentation)

That said, please let me know if I misunderstood the purpose of this suggestion.

Reply:
Thank you a lot for sharing that and help me indeed improve the wording skills !!!